velox/common/compression/CMakeLists.txt

```diff
@@ -16,8 +16,13 @@ if(${VELOX_BUILD_TESTING})
   add_subdirectory(tests)
 endif()
 
-velox_add_library(velox_common_compression Compression.cpp LzoDecompressor.cpp)
+velox_add_library(
+  velox_common_compression
+  Compression.cpp
+  LzoDecompressor.cpp
```

Comment:
see my comment above. It would be great if we could compile these things in a more modular way (only enabling the compression methods that were available at compilation time)